src/iuse.cpp

```diff
@@ -7490,48 +7490,80 @@ int iuse::ladder( player *p, item *, bool, const tripoint& )
     return 1;
 }
 
+
 int iuse::washclothes( player *p, item *it, bool, const tripoint& )
 {
+    // Check that player isn't over volume limit as this might cause it to break... this is a hack.
+    // TODO: find a better solution.
+    if (p->volume_capacity() < p->volume_carried()){
```

Comment:
Could be astyled.

Reply:
This should look like `if( p->volume_capacity() < p->volume_carried() ) {` (spaces inside the brackets and space before the curly bracket).